### PR TITLE
Support Python 3.5

### PR DESCRIPTION
**Pull request by myint**
This involves fixing a test and avoiding a function deprecated since Python 3.0.

https://docs.python.org/dev/library/inspect.html#inspect.getargspec

### CODE REVIEW

**Comment by myint** [comment]
By the way, I think it would be a good idea to enable Travis on pull requests for this repository. Currently pull requests are not tested for some reason. I think it is a Travis CI setting.

**Comment by sigmavirus24** [comment]
I toggled Travis through its control pane (off then on) to re-enable this. Good catch @myint

**Comment by sigmavirus24** [comment]
Can you also add the `- nightly` to this pull request?

**Comment by sigmavirus24** [comment]
`inspect.signature` does not exist on Python 3.2. It looks as though the [deprecation notice for Python 3.0](https://docs.python.org/3.2/library/inspect.html#inspect.getargspec) says to use `inspect.getfullargspec` instead of `inspect.signature`. Even so, we can use signature (it seems) on 3.3 and beyond.

File: pep8.py [hunk 1]
```diff
@@ -1328,7 +1328,12 @@ def _add_check(check, kind, codes, args):
                 codes = ERRORCODE_REGEX.findall(check.__doc__ or '')
             _add_check(check, args[0], codes, args)
     elif inspect.isclass(check):
-        if inspect.getargspec(check.__init__)[0][:2] == ['self', 'tree']:
+        if sys.version_info[0] >= 3:
```
Comment (sigmavirus24) [inline pep8.py]:
```py
if sys.version_info >= (3, 3):
```

That will suffice for what we need. `inspect.signature` was introduced in 3.3 and not before.

**Comment by myint** [comment]
Should be fixed now.

**Comment by sigmavirus24** [comment]
This looks good to me. :+1:

**Comment by IanLee1521** [comment]
Merged! Thanks!

**Comment by SylvainDe** [comment]
Thanks for fixing this issue! :+1: 
Out of curiosity, does anyone know if this is going to be part of the next pep8 release ? If so, is there any expected release date ?

**Comment by myint** [comment]
I would be interested in a PyPI release as well.

**Pull request merged**